src/pages/iou/request/IOURequestStartPage.tsx

```diff
@@ -96,6 +96,12 @@ function IOURequestStartPage({
         if (transaction?.reportID === reportID) {
             return;
         }
+        // if transaction is from global create, the original behavior is that the user can change the recipient reportID
+        // without having to reset the whole transaction, hence we don't need to re-init the transaction in this case
+        if (transaction?.isFromGlobalCreate && transaction.transactionID) {
+            IOU.resetMoneyRequestReportID(transaction.transactionID, reportID, isFromGlobalCreate);
+            return;
+        }
```

Comment:
We don't need this anymore. Now the back button uses the transaction's report id

Reply:
When navigating back to the participants page we will use the transaction's report id in the uri param. Thus the condition above `transaction?.reportID === reportID` will be met and we won't init the money request.

Reply:
In `navigateToStartStepIfScanFileCannotBeRead` can we navigate to the scan step?

Reply:
Currently the user is being navigated to the start step (`ROUTES.MONEY_REQUEST_CREATE_TAB_SCAN`). There is another page where the user can only select the receipt (`ROUTES.MONEY_REQUEST_STEP_SCAN`). Can we navigate to that page?